frontend/components/DiskSpaceGraph/_styles.scss

```diff
@@ -0,0 +1,45 @@
+.info-flex,
```

Comment:
Is it necessary to modify the `.info-flex` class here?

Reply:
It's needed for the & selector, info-flex is the classname used on device page and host details page, but I wanted to use gigs_disk_space_available__cell for the manage host page to match the class names there, hence the two classes

Reply:
That makes sense :) 

My only hesitation was that by having it open-ended (because of the comma on line 1) someone might inadvertently add a new style that ends up conflicting with the general `.info-flex`. For example, if you add `background-color: red` on between line 4 and 5 or between line 44 and 45, it will affect other the `.info-flex` selectors on the host details page. Admittedly, that's an extreme example, but I think it might be worth trying to using `.info-flex__disk-space` and `info-flex.disk-space-tooltip` to keep it as narrow as possible.